modules/library/jdbc/src/test/java/org/geotools/data/jdbc/FilterToSQLTest.java

```diff
@@ -67,7 +66,7 @@
 public class FilterToSQLTest extends TestCase {
     private FilterFactory filterFac = CommonFactoryFinder.getFilterFactory(null);
     private static Logger LOGGER = org.geotools.util.logging.Logging.getLogger("org.geotools.data.jdbc");
-    
+
```

Comment:
Seems it's just whitespace change, please avoid reformatting the code in areas that you did not touch (automatic formatting and checking is coming, please be patient, it's linked to my very limited spare time)